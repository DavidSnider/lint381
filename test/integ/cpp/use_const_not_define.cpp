#ifndef FOO_H
#define FOO_H

#define FOO (10)

#define BAR "bar"

#define BAZ(QUX) GRAULT
