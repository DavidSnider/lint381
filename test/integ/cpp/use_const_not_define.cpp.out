use_const_not_define.cpp:4:1: error: Use 'const' or 'constexpr' to create constant 'FOO', not '#define'
#define FOO (10)
^^^^^^^^^^^^^^^^
use_const_not_define.cpp:6:1: error: Use 'const' or 'constexpr' to create constant 'BAR', not '#define'
#define BAR "bar"
^^^^^^^^^^^^^^^^^
